support-workers/src/test/resources/logback.xml

```diff
@@ -1,6 +1,6 @@
 <configuration>
 
-    <appender name="STDOUT" class="io.symphonia.lambda.logging.DefaultConsoleAppender">
```

Comment:
If we are no longer using the symphonia library we can remove it from the dependencies. Have you tested that this works ok with writing to CloudWatch?

Reply:
this is the "src/test/" version, the normal one is still the same.